install/kubernetes/cilium/templates/cilium-agent-daemonset.yaml

```diff
@@ -372,7 +372,6 @@ spec:
           capabilities:
             add:
             - NET_ADMIN
-          privileged: true
```

Comment:
maybe we can have a quick look for this secion https://github.com/cilium/cilium/blame/master/Documentation/operations/system_requirements.rst#L387-L396 and check if any change is required.

Reply:
Good spotting, yes we are looking into the capabilities that Cilium needs. @borkmann the docs says `CAP_SYS_ADMIN` is needed but Cilium only have `NET_ADMIN`, does `NET_ADMIN` also have `SYS_ADMIN`?